cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -460,34 +460,27 @@ export class Assessment extends Application {
         }
     }
 
-    openCopyAssessmentModel(): void {
-        Assessment.open();
-        selectItemsPerPage(100);
-        cy.wait(2000);
-        cy.get(tdTag)
-            .contains(this.name)
-            .parent(tdTag)
-            .parent(trTag)
-            .within(() => {
-                click(actionButton);
-                cy.wait(500);
-                clickByText(button, "Copy assessment");
-            });
-    }
-
-    openCopyAssessmentReviewModel(): void {
+    openCopyAssessmentModel(review = false): void {
+        let action = "";
+        if (review) {
+            action = "Copy assessment and review";
+        } else {
+            action = "Copy assessment";
+        }
```

Comment:
**Minor suggestion:**

    let action = "Copy assessment";
        if (review) {
            action += " and review";
        }

Reply:
Done